types/esbuild-vue/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitThis": true,
+        "noImplicitAny": true,
+        "strictFunctionTypes": true,
+        "esModuleInterop": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
```

Comment:
Why have you removed `esModuleInterop`? See the bot's comment.

Reply:
Hmm, but it's [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67017/commits/7a6349baa3d2683bb9a2b4c9eed0082452bf718f#diff-99064c37cfd03a0fde18694cb8b3297ca22876636bf6c6921d26c658e211ad77R10).

Reply:
Hm, the bot's still unhappy with it, for some reason. Maybe it needs to be gone entirely?

Reply:
That's right. It's okay now.